tests/test_ai.py

```diff
@@ -0,0 +1,59 @@
+"""This module contains the tests for the SUSE AI container."""
+
+import pytest
+import requests
+from tenacity import retry
+from tenacity import stop_after_attempt
+from tenacity import wait_exponential
+
+from bci_tester.data import OLLAMA_CONTAINER
+from bci_tester.data import OPENWEBUI_CONTAINER
+
+CONTAINER_IMAGES = (
+    OLLAMA_CONTAINER,
+    OPENWEBUI_CONTAINER,
+)
+
+
+@pytest.mark.parametrize(
+    "container_per_test",
+    [OLLAMA_CONTAINER],
+    indirect=["container_per_test"],
+)
+def test_ollama_health(container_per_test):
+    """Test that we can reach the port 11434 successfully."""
+    host_port = container_per_test.forwarded_ports[0].host_port
+
+    # Retry 5 times with exponential backoff delay
+    @retry(
+        wait=wait_exponential(multiplier=1, min=4, max=10),
+        stop=stop_after_attempt(5),
+    )
+    def check_ollama_response():
+        resp = requests.get(f"http://localhost:{host_port}/", timeout=30)
+        resp.raise_for_status()
+        assert "Ollama is running" in resp.text
+
```

Comment:
For now OK, I suggest to add another test that performs a query against ollama and not only check if the default page works.